examples/init_api_client.py

```diff
@@ -30,12 +30,20 @@ def get_onprem_api_client(args):
 
     logger.info("Getting api client for IP <{}>".format(args.platform_ip))
     api_client = swagger_client.ApiClient(host="https://{}/api/ni".format(args.platform_ip))
-    user_creds = swagger_client.UserCredential(username=args.username, password=args.password,
-                                               domain=dict(domain_type=args.domain_type))
-
     auth_api = swagger_client.AuthenticationApi(api_client=api_client)
-
-    auth_token = auth_api.create(user_creds)
+    if args.domain_type == "LOCAL" or args.domain_type == "LDAP":
+        user_creds = swagger_client.UserCredential(username=args.username, password=args.password,
+                                                   domain=dict(domain_type=args.domain_type))
+        auth_token = auth_api.create(user_creds)
+    elif args.domain_type == "VIDM":
+        if args.get_vidm_client_id:
+            client_id = auth_api.get_vidm_oauth_clien_id()
```

Comment:
is this fn name corrent ? get_vidm_oauth_clien_id

Reply:
yes its correct.
 i noticed just "t" is missing in swagger as well.

![image](https://user-images.githubusercontent.com/20511903/78247672-e4fa0480-7508-11ea-86d6-1337613b652e.png)